src/placeos-models/asset_instance.cr

```diff
@@ -0,0 +1,47 @@
+require "./base/model"
+require "./user"
+
+module PlaceOS::Model
+  class AssetInstance < ModelBase
+    include RethinkORM::Timestamps
+
+    enum Tracking
+      InStorage
+      OnTheWay
+      InRoom
+      Returned
+    end
+
+    table :ass
+
+    attribute tracking : Tracking = Tracking::InStorage
+    attribute approval : Bool = false
+    attribute requester_id : String?
+
+    attribute duration_start : Time
+    attribute duration_end : Time
```

Comment:
What is `duration_start` and `duration_end`? A booking?

Reply:
I think that is the time the asset is in use for, when it's in a room